apps/angular/simple-animations/src/app/app.component.ts

```diff
@@ -17,9 +25,32 @@ import { Component } from '@angular/core';
       }
     }
   `,
+  animations: [
+    trigger('slideIn', [
+      transition(':enter', [
+        style({ transform: 'translateX(-100%)' }),
+        animate('1000ms ease-in', style({ transform: 'translateX(0%)' })),
```

Comment:
Maybe it's possible to use `transform: '*'` instead of `transform: 'translateX(0%)'`. 
More information about the `*` token [here](https://angular.io/api/animations/style).

Reply:
Yes, `auto-styling` works here.  `auto-styling` gives more flexibility. If the `transform` were to change, you wouldn't have to update the `animate`.   I'll leave the code as it is, but I don't know any negatives to using `auto-styling`.